pkgs/applications/gis/qgis/unwrapped.nix

```diff
@@ -120,36 +120,50 @@ in mkDerivation rec {
     ++ lib.optional withWebKit qtwebkit
     ++ pythonBuildInputs;
 
-  nativeBuildInputs = [ makeWrapper wrapGAppsHook cmake flex bison ninja ];
+  nativeBuildInputs = [ makeWrapper wrapQtAppsHook cmake flex bison ninja ];
```

Comment:
Are we sure that `wrapGAppsHook` is really not needed ? See comment above.

Reply:
No I think I should probably revert this.